helm/templates/minio/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+{{- if .Values.minio.enabled }}
```

Comment:
can we just use the minio helm chart as a dependency? 
https://github.com/minio/charts (probably this is the old one)

Reply:
Yes that should work. I think I actually looked it up and didn't add it because the repo was archived. But minio is only for sandbox anyway right?

Reply:
Yup it's only sandbox- I am ok keeping it too